python3: error while loading shared libraries: libpython3.9.so.1.0: cannot open shared object file: No such file or directory
python3: error while loading shared libraries: libpython3.9.so.1.0: cannot open shared object file: No such file or directory
TERRIBLE_ERROR_HAPPENED
